packages/loader/container-loader/src/container.ts

```diff
@@ -1316,13 +1306,11 @@ export class Container
 	private async load(
 		specifiedVersion: string | undefined,
 		loadMode: IContainerLoadMode,
+		resolvedUrl: IResolvedUrl,
```

Comment:
This change to pass the resolved url in the load call rather than grabbing it in the constructor seems very positive regardless of whether you keep the resolvedUrl member - can you remove `IContainerConfig.resolvedUrl` from the interface entirely?